Phpdoc is required only for traversable types (#5)
Swivl/php-coding-standard: README.md

    </properties>
</rule>
```
* `concreteTypeToBaseTypeMap` - when column is mapped to concrete type FeedItemComment, but modification methods are type hinted with base class CommentInterface
* `useDynamicalCalculationForEnumColumnType` - when enum is mapped to some PHP type.
For example, `type="auth_enum_type"` is mapped to PHP AuthType.

> #### `Swivl.Commenting.FunctionComment`

Reviewer: Please add single empty line before this header line.

This sniff provides the option:
```xml
<rule ref="Swivl.Commenting.FunctionComment">
    <properties>
        <property name="requiredPhpdoc" value="always"/>
    </properties>
</rule>
```

> * `requiredPhpdoc` - is responsable for which type of methods PHPDoc is requaired. Posible values:
>     * `always` - PHPDoc is requaired for all methods.
>     * `map` - PHPDoc is requaired for a method if at least 1 type hint or return type is the list which is defined in the sniff in FunctionCommentSniff::REQUIRED_PHPDOC_MAP. It is default behavior.
>     * `never` - PHPDoc isn't requaired for all methods.

Reviewer: ```suggestion
* `requiredPhpdoc` - configures when PHPDoc is required for a method. Possible values:
    * `always` - PHPDoc is required for all methods.
    * `map` - PHPDoc is required for a method if at least 1 type hint or return type is in the list which is defined in the sniff in FunctionCommentSniff::REQUIRED_PHPDOC_MAP. It is the default behavior.
    * `never` - PHPDoc isn't required for all methods.
```

Dependencies update
-------------------
Some sniffs from Swivl Coding Standard override sniffs from libraries which are in dependencies list.

> In list below are sniffs which need to be checked when dependensies are updating:

Reviewer: ```suggestion
The following sniffs should be synchronized with the original code during dependencies update:
```

* [Swivl\Sniffs\Commenting\FunctionCommentSniff](blob/master/Swivl/Sniffs/Commenting/FunctionCommentSniff.php)
* [Swivl\Sniffs\Functions\MultiLineFunctionDeclarationSniff](blob/master/Swivl/Sniffs/Functions/MultiLineFunctionDeclarationSniff.php)

> * [Swivl\Sniffs\Namespaces\NamespaceDeclarationSniff](blob/master/Swivl/Sniffs/Namespaces/NamespaceDeclarationSniff.php)

Reviewer: These links are broken in the rich representation, please fix.
